pages/common/git-sed.md

```diff
@@ -0,0 +1,22 @@
+# git sync
+
+> Run grep as directed but replace the given files with the pattern.
+> Works only on git tracked files.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-sed>.
+
+- Replace the string 'this' with 'that':
+
+`git sed 'this' 'that'`
+
+- Replace the string and then commit the resulting changes with a standard commit message:
+
+`git sed -c 'this' 'that'`
+
+- Replace the string passing regex flags to `sed` :
+
+`git sed -f g 'this' 'that'`
```

Comment:
The docs doesn't use `-f` to use the `g`

Reply:
Yes, I noticed it. I was looking at the man page and there's mention of using `-f g`

Not sure if I should follow the man page or the docs
![image](https://user-images.githubusercontent.com/8644691/139695493-1d28729a-f034-420b-8238-e47f43d41ee7.png)

Reply:
I think as long as it works it's not important.